soccer/Robot.cpp

```diff
@@ -190,58 +195,56 @@ void OurRobot::move(Geometry2d::Point goal, Geometry2d::Point endVelocity) {
         cout << " in OurRobot::move(goal): adding a goal (" << goal.x() << ", "
              << goal.y() << ")" << std::endl;
 
-    intent().motion_command = std::make_unique<Planning::PathTargetCommand>(
-        MotionInstant(goal, endVelocity));
+    Planning::RobotInstant goal_instant;
+    goal_instant.pose = Pose{goal, 0};
+    goal_instant.velocity = Twist{endVelocity, 0};
+    setMotionCommand(std::make_unique<MotionCommand>(
+        Planning::PathTargetCommand{goal_instant}));
 
     *_cmdText << "move(" << goal.x() << ", " << goal.y() << ")" << endl;
     *_cmdText << "endVelocity(" << endVelocity.x() << ", " << endVelocity.y()
               << ")" << endl;
 }
 
-void OurRobot::settle(std::optional<Point> target) {
+void OurRobot::settle() {
     if (!visible()) return;
-
-    intent().motion_command = std::make_unique<Planning::SettleCommand>(target);
```

Comment:
FWIW, settle with target was filling two roles in my mind when I made it. A first run of a one touch command results in almost the exact same code in terms of finding where to intercept, getting there, and then stopping. the only difference is you want to face a certain direction and kick on breakbeam. The second role is that if you want to two touch settle a ball, you would settle at an angle and allow the not fully damped ball bounce in the direction of the next shot/pass and one touch the slow moving ball.

I haven't looked at the rest of the code to make sure one touch still worked in this new system.